gnovm/pkg/doc/dirs.go

```diff
@@ -0,0 +1,163 @@
+// Mostly copied from go source at tip, commit d922c0a.
```

Comment:
Is this file really copied? I was not able to find it in the [Go source code](https://github.com/golang/go/tree/d922c0a8f5035b0533eb6e912ffd7b85487e3942/src/go/doc/comment). Could you add the path to the file too?

Reply:
Ah, it's part of a [different package](https://github.com/golang/go/blob/d922c0a8f5035b0533eb6e912ffd7b85487e3942/src/cmd/doc/dirs.go). I'll add a link :)